doc/api/documentation.md

```diff
@@ -32,6 +32,18 @@ The stability indices are as follows:
 > [semantic versioning][] rules. Non-backward compatible changes or removal may
 > occur in any future release. Use of the feature is not recommended in
 > production environments.
+>
+> Experimental features are subdivided into stages:
+>
+> * 1.0 - Early development. Experimental features at this stage are unfinished
+>   and subject to substantial change.
```

Comment:
```suggestion
>   and subject to substantial change or removal.
```

Reply:
I already talk about expected level of change though. I think at no stage do we expect a feature to be removed, though of course it can happen and we say so above this list.